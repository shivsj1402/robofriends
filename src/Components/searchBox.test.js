import React from "react";
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import SearchBox from "./searchBox";

it('searchBox component render', ()=>{
    expect(toJson(shallow(<SearchBox />))).toMatchSnapshot();
})
