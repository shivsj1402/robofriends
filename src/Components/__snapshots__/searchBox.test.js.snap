// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`searchBox component render 1`] = `
<div
  className="pa2"
>
  <input
    className="pa3 ba b--green bg-lightest-blue"
    placeholder="serach robots"
    type="search"
  />
</div>
`;
